api/service.proto

```diff
@@ -35,6 +35,24 @@ message CreateArtifactTypeResponse {
   ml_metadata.ArtifactType artifact_type = 1;
 }
 
+message PutTypeOptions {
```

Comment:
Can we rename PutTypeOptions to `UpdateTypeOptions`? `Put` is too specific to HTTP verb.

But I don't think we need it. See my comment below.

Reply:
We shall expose it later. But the update function does not expose the option looks a bit weird for now.